src/IssueViz.UnitTests/Commands/IssueVisualizationToolWindowCommandTests.cs

```diff
@@ -127,75 +128,53 @@ public void ErrorListQueryStatus_CommandVisibilityIsSetToContextActivation(bool
         }
 
         [TestMethod]
-        public void Execute_WindowNotCreated_NoException()
+        public void Execute_ServiceCalled()
```

Comment:
Same as the other PR: the old tests are redundant.
The new tests were cut and paste from the PR and the names of the types being tested etc were changed. Other than that the tests are the same as the other PR.

I've manually tested these changes too.